app/Http/Controllers/QuizController.php

```diff
@@ -34,4 +34,19 @@ public function instructions(Quiz $quiz)
 
         return view('quizzes.instructions', compact('quiz'));
     }
+
+    public function getVerify(Quiz $quiz)
```

Comment:
extract these two methods into a seperate controller, probably call it `QuizVerificationController` and the methods could be `showVerifyForm` and `verify`